pkg/utils/image_test.go

```diff
@@ -40,12 +40,15 @@ var _ = Describe("Image retrieval tests", func() {
 	It("etcd spec defines etcd and backup-restore images", func() {
 		etcd = testutils.EtcdBuilderWithDefaults(etcdName, namespace).Build()
 		imageVector = createImageVector(true, true, false, false)
-		etcdImage, etcdBackupRestoreImage, err := GetEtcdImages(etcd, imageVector, false)
+		etcdImage, etcdBackupRestoreImage, alpine, err := GetEtcdImages(etcd, imageVector, false)
 		Expect(err).To(BeNil())
 		Expect(etcdImage).ToNot(BeNil())
 		Expect(etcdImage).To(Equal(etcd.Spec.Etcd.Image))
 		Expect(etcdBackupRestoreImage).ToNot(BeNil())
 		Expect(etcdBackupRestoreImage).To(Equal(etcd.Spec.Backup.Image))
+		vectorAlpineImage, err := imageVector.FindImage(common.Alpine)
```

Comment:
similar change is required for `vectorAlpineImage`

Reply:
Done